src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -1068,9 +1068,9 @@ private static void RemoveEventHandler(object key, Delegate? value)
 
             lock (s_eventLockObject)
             {
-                if (s_handlers != null && s_handlers.ContainsKey(key))
+                if (s_handlers != null && s_handlers.TryGetValue(key, out var val))
                 {
-                    List<SystemEventInvokeInfo> invokeItems = s_handlers[key];
+                    List<SystemEventInvokeInfo> invokeItems = val;
```

Comment:
This extra temporary isn't necessary... you can delete this line and just rename `val` above to be `invokeItems`, e.g. `out List<SystemEventInvokeInfo>? invokeItems`.